src/utils/init-project/config.yaml

```diff
@@ -0,0 +1,25 @@
+# ╭──────────────────────────────────────ImplementationGuide───────────────────────────────────────╮
+# │  The properties below are used to create the ImplementationGuide resource. For a list of       │
+# │  supported properties, see:                                                                    │
+# │  http://build.fhir.org/ig/HL7/fhir-shorthand/branches/beta/sushi.html#ig-development           │
```

Comment:
And yet another thing you did exactly as I told you but is probably wrong now.  I think we need to change this to the new location: https://fshschool.org/sushi/configuration/

In fact, we probably need to change this in other parts of SUSHI too (because we produce a similar message to this when we generate config files from package.json.